libfreerdp/core/gateway/wst.c

```diff
@@ -641,11 +715,17 @@ static BOOL wst_parse_url(rdpWst* wst, const char* url)
 
 	if (strncmp("wss://", url, 6) != 0)
 	{
-		WLog_ERR(TAG, "Websocket URL is invalid. Only wss:// URLs are supported");
-		return FALSE;
+		if (strncmp("https://", url, 8) != 0)
```

Comment:
Shall we always relax the check for `wss://` or is it AMD specific ?

Reply:
I'll update the message.

Reply:
My remark was not for the message but for the check itself. Do we want to always relax the check for all websocket things, or just for AMD ?